src/Utils/request/utils.ts

```diff
@@ -50,33 +50,24 @@ const ensurePathNotMissingReplacements = (path: string) => {
   }
 };
 
-export function makeHeaders(noAuth: boolean) {
+export function makeHeaders(noAuth: boolean, additionalHeaders?: HeadersInit) {
   const headers = new Headers({
     "Content-Type": "application/json",
     Accept: "application/json",
+    ...additionalHeaders,
   });
 
   if (!noAuth) {
-    const token = getAuthorizationHeader();
+    const token = localStorage.getItem(LocalStorageKeys.accessToken);
 
     if (token) {
-      headers.append("Authorization", token);
+      headers.append("Authorization", `Bearer ${token}`);
```

Comment:
If we set the token earlier and then set the new headers (so overwriting it in case we are passing in an additionalHeader, that would be enough, no?

We can then remove the customHeader in queryRequest

@rithviknishad

Reply:
That's fine as well. What do you think? @rithviknishad

Reply:
Yup